app/src/main/java/app/rive/runtime/example/LowLevelActivity.kt

```diff
@@ -33,7 +33,14 @@ class LowLevelRiveView(context: Context) : RiveTextureView(context) {
     // Initialize renderer first: we can't create Files without one.
     override val renderer = object : RendererSkia() {
         override fun draw() {
-            artboard.drawSkia(cppPointer, Fit.COVER, Alignment.CENTER)
+            artboard?.let {
+                save()
+                align(Fit.COVER, Alignment.CENTER,  AABB(width,height), it.bounds)
+                it.drawSkia(
+                    cppPointer
+                )
+                restore()
+            }
```

Comment:
Mmh is it ok to ask our users to do this when running your own loop? 
Maybe it is! Otherwise, we can keep the old call around in the artboard API for ease of use?